kaolin/cython/mcubes.cpp

```diff
@@ -891,7 +877,7 @@ static const char *__pyx_f[] = {
   "type.pxd",
 };
 
-/* "../../.local/lib/python3.6/site-packages/Cython/Includes/numpy/__init__.pxd":776
+/* "../../../miniconda3/envs/temptemp37/lib/python3.7/site-packages/Cython-0.29.14-py3.7-linux-x86_64.egg/Cython/Includes/numpy/__init__.pxd":776
```

Comment:
I see that there is a lot of those diff in your PR, can you explain what is it doing ? (It seems commented, right ?)

Reply:
Hmm, these files are generated by Cython, so I didn't dig into them You're right, I'll go through and clean them up.

Reply:
If the files are generated by Cython, should we actually put them in the repository ? Shouldn't `setup.py` generate them for the user ?

Reply:
See the first point in the description.

Reply:
Comments (original source code) are now omitted from the generated C files.